src/cards/ha-entities-card.html

```diff
@@ -91,7 +93,19 @@
     return classes;
   }
 
+  addTapEvents() {
+    var cards = this.root.querySelectorAll('.has-more-card');
+    for (var i of cards) {
+      i.addEventListener('tap', this.entityTapped);
+    }
+  },
+
+  computeStateClass(stateObj) {
+    return (window.hassUtil.stateMoreInfoType(stateObj) !== 'hidden' ? 'state has-more-card' : 'state');
+  },
+
   entityTapped(ev) {
+    alert(ev.target.nodeName + "\n" + ev.model);
```

Comment:
ev.model is undefined, how can I solve this?

Reply:
...Or we use [this (working)](https://github.com/home-assistant/home-assistant-polymer/compare/master...c727:patch-5). Any feedback would be great